pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -184,6 +184,17 @@ type ShootStatus struct {
 	// managed to change the hibernation settings of the cluster
 	// +optional
 	LastHibernationTriggerTime *metav1.Time `json:"lastHibernationTriggerTime,omitempty" protobuf:"bytes,17,opt,name=lastHibernationTriggerTime"`
+	// LastMaintenanceOperation holds information about the last maintenance operation on the Shoot.
+	// +optional
+	LastMaintenanceOperation *LastMaintenanceOperation `json:"lastMaintenanceOperation,omitempty" protobuf:"bytes,18,opt,name=lastMaintenanceOperation"`
```

Comment:
```suggestion
	LastMaintenance *LastMaintenance `json:"lastMaintenance,omitempty" protobuf:"bytes,18,opt,name=lastMaintenance"`
```

Change this consistently everywhere